src/logic/player.h

```diff
@@ -255,11 +253,10 @@ class Player {
 		/// has ever seen it.
 		///
 		/// The value is
-		///  0    if the player has never seen the node
-		///  1    if the player does not currently see the node, but has seen it
-		///       previously
-		///  1+n  if the player currently sees the node, where n is the number of
-		///       objects that can see the node.
+		///  `kUnexplore` if the player has never seen the node
+		///  `kUnsee`     if the player does not currently see
```

Comment:
How about renaming these to `kUnexplored`, `kPreviouslySeen` and `kRevealed`? I'd find that easier to understand.

Reply:
Previously these were used only as command names to pass to `hide_or_reveal_fields()`. I'll update the names.

Reply:
Thanks! Can you align the comments again? From

```
		///  `kUnexplored` if the player has never seen the node
		///  `kPreviouslySeen`     if the player does not currently see
		///               the node, but has seen it previously
		///  `kVisible`    if the player currently sees the node
```

to

```
		///  `kUnexplored`      if the player has never seen the node
		///  `kPreviouslySeen`  if the player does not currently see
		///                     the node, but has seen it previously
		///  `kVisible`         if the player currently sees the node
```